fuzzydir.lua

```diff
@@ -20,18 +20,38 @@
     Determines the max depth of recursive search, should be >= 1
 
     Examples for "sub-file-paths = **":
-    "max_search_depth = 1" => finds [xyz.ass, subs/xyz.ass]
-    "max_search_depth = 2" => finds [xyz.ass, subs/xyz.ass, subs/moresubs/xyz.ass]
+    "max_search_depth = 1" => finds [subs/xyz.ass]
+    "max_search_depth = 2" => finds [subs/xyz.ass, subs/moresubs/xyz.ass]
 
     Please be careful when setting this value too high as it can result in awful performance or even stack overflow
 ]]
-local max_search_depth = 3
 
-local utils = require "mp.utils"
+local msg = require 'mp.msg'
+local options = require 'mp.options'
+local utils = require 'mp.utils'
+
+o = {
+    max_search_depth = 3, --determines the max depth of recursive search, should be >= 1
```

Comment:
I don't think there's any reason to duplicate documentation for properties as there is a dedicated comment block just for this purpose.

Reply:
Done